.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,93 @@
+name: E2E - Run tests
+
+on:
+  workflow_dispatch:
+  # push:
```

Comment:
do you want to only run on `workflow_dispatch` for now until having all the ests ready?